pkg/sql/logictest/tests/fakedist/fakedist_test: TestLogic_buffered_writes failed

pkg/sql/logictest/tests/fakedist/fakedist_test.TestLogic_buffered_writes [failed](https://tanzanite.cluster.engflow.com/invocations/default/776c60fc-5c94-4b60-8f93-2594879f8678?testReportRun=2&testReportShard=33&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9mYWtlZGlzdDpmYWtlZGlzdF90ZXN0) on release-25.2.0-rc @ [545da2936c70fdecb9bbbd652aa4bd90b8d26dad](https://github.com/cockroachdb/cockroach/commits/545da2936c70fdecb9bbbd652aa4bd90b8d26dad):


```
=== RUN   TestLogic_buffered_writes
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogic_buffered_writes1811528508
    test_log_scope.go:76: use -show-logs to present logs inline
    logic.go:1570: apply cluster opt logictest.clusterOptDisableUseMVCCRangeTombstonesForPointDeletes
    test_server_shim.go:141: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
[11:35:51] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/buffered_writes with config fakedist: 80 tests, 1 failures
[11:35:58] --- total progress: 80 statements
--- total: 80 tests, 1 failures
    logic.go:4510: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_buffered_writes1811528508
--- FAIL: TestLogic_buffered_writes (73.49s)
=== RUN   TestLogic_buffered_writes/regression
[11:35:00] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/buffered_writes: 69 statements
[11:35:19] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/buffered_writes: 76 statements
[11:35:36] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/buffered_writes: 79 statements
[11:35:50] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/buffered_writes: 80 statements
    logic.go:2307: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/buffered_writes:449: 
        expected success, but found
        (40001) restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE): "sql txn" meta={id=3124773b key=/Table/116/1/111/0 iso=Serializable pri=0.00536814 epo=0 ts=1746099320.603981844,2 min=1746099319.193831193,0 seq=5} lock=true stat=PENDING rts=1746099319.193831193,0 wto=false gul=1746099319.693831193,0 obs={n1@1746099319.193831193,0}
--- FAIL: TestLogic_buffered_writes/regression (53.59s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=33</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_buffered_writes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-50271

--- a/pkg/sql/logictest/testdata/logic_test/buffered_writes
+++ b/pkg/sql/logictest/testdata/logic_test/buffered_writes
@@ -404,7 +404,7 @@ SELECT count(*) FROM uvw
 user host-cluster-root
 
 statement ok
-SET CLUSTER SETTING kv.closed_timestamp.target_duration = '30s'
+SET CLUSTER SETTING kv.closed_timestamp.target_duration = '99999s'
 
 user root
 

